src/Bravo3/ImageManager/Services/ImageManager.php

```diff
@@ -170,6 +186,38 @@ public function pull(Image $image)
     }
 
     /**
+     * Get meta information about the source image from the cache layer.
+     *
+     * @param Image $image
+     *
+     * @return ImageMetadata
+     */
+    public function getMetadata(Image $image)
+    {
+        $img_key = $image->getKey();
```

Comment:
Don't gratuitously do things you don't need:

        // If the $image is a variation, refer to its parent for the metadata.
        if ($image instanceof ImageVariation) {
            $img_key = $image->getKey(true);
        } else {
            $img_key = $image->getKey();
        }